optimum/exporters/openvino/model_configs.py

```diff
@@ -1610,12 +1613,49 @@ def patch_model_for_export(
         self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
     ):
         model_kwargs = model_kwargs or {}
-        if self._behavior != LlavaConfigBehavior.VISION_EMBEDDINGS:
+        if self._behavior != VLMConfigBehavior.VISION_EMBEDDINGS:
+            return super().patch_model_for_export(model, model_kwargs)
+        return CommonImageEmbeddingsModelPatcher(self, model, model_kwargs)
+
+
+class VLMOpenVINOConfig(BaseVLMOpenVINOConfig):
```

Comment:
having both `VLMOpenVINOConfig` and `BaseVLMOpenVINOConfig` is confusing, what's the difference ?

Reply:
remains only one